evm/vm/forks/sharding/log_handler.py

```diff
@@ -117,3 +123,7 @@ def get_new_logs(self, address=None, topics=None):
             topics,
         )
         return self.w3.eth.getLogs(filter_params)
+
+    def reset(self):
```

Comment:
This function is at least not used at all in the changelog of this pull request.  

Also, I find it odd that it resets to genesis.  Intuitively I would expect it to reset to it's original state (as in the state it was in when instantiated, or maybe just the most recent blockhashes).  Maybe rename to `reset_to_genesis`?

Reply:
So sorry for this. You're right it is indeed not used anymore. I think I can just remove it.
Thank you so much for pointing it out.